src/bindings/python/config.py

```diff
@@ -0,0 +1,50 @@
+import json
+import os
+
+from .valhalla_build_config import config as _config, help_text as _help_text, optional as _optional
+
+_global_config = dict()
```

Comment:
once configured we don't have to rely on the c++ property tree anymore for doing simple things like tarring up tiles..